app/models/user.rb

```diff
@@ -659,5 +659,4 @@ def setup_pro_account(role)
   def update_pro_account
     pro_account.update_stripe_customer if pro_account
   end
-
```

Comment:
Move from 9e0485dac6c651e667862764a31a462df8128547 to 86fc1801a011f1ab0bf47b805cf2f52219ab5e6f

Reply:
Oops, fixed.